static/js/common/upload-addon.js

```diff
@@ -27,7 +27,7 @@
     }
 
     $.fn.addonUploader = function( options ) {
-        var settings = {'filetypes': ['xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
+        var settings = {'filetypes': ['zip', 'xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
```

Comment:
`.json` was added for webapps https://bugzilla.mozilla.org/show_bug.cgi?id=676887